core_backend/app/utils.py

```diff
@@ -1,12 +1,22 @@
 import hashlib
 import logging
+import secrets
 from logging import Logger
 
 import aiohttp
 
 from .config import LOG_LEVEL
 
 
+def generate_key() -> str:
+    """
+    Generate a new 32-character API key
+    """
+
+    # 24 bytes gives 32 characters here
+    return secrets.token_urlsafe(24)
```

Comment:
Should we use 32 bytes?

https://docs.python.org/3/library/secrets.html#how-many-bytes-should-tokens-use

Reply:
Yeah that makes sense given the link. 32 bytes makes 43 characters which was an ugly number but its ok haha